portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -40,11 +66,23 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String comment = getParameter(request, "text-input", "");
+    String name = getParameter(request, "name-input", "");
+    String message = getParameter(request, "text-input", "");
+    long timestamp = System.currentTimeMillis();
+
+    Date date = new Date(timestamp);
+    final SimpleDateFormat dateFormat = new SimpleDateFormat("MM/dd/yyyy");
```

Comment:
I see, right, you could not put `static final` inside an instance method. I meant you can do 
`private static final SimpleDateFormat dateFormat = new SimpleDateFormat("MM/dd/yyyy");` at class level **outside** of the method `doPost()`.